### PR TITLE
Adds Ammo Boxes to the outpost console.

### PR DESCRIPTION
**Pull request by Martinpachu**
## About The Pull Request
Adds ammo boxes to the cargo console,allowing you to buy them instead of just mags. Adds common pistol calibers and 7.62. Also changes bought shotgun boxes to the ammo_box version

## Why It's Good For The Game
Buying magazines is very overpriced and fills your ship with useless empty mags if you want to rely on cargo for your ammo,and 7.62 boxes are unobtainable anywhere else. I would've added other calibers of cargo guns like 4.6 but they don't have ammo box versions. Also shotgun ''ammo'' boxes are annoying.
## Changelog

:cl:
add: Ammo boxes to the outpost market.
spellcheck: Changed AK mentions to SVG.
tweak: Changed shotgun ammo boxes bought from the outpost to the ammo_box version.
fix: Changed SVG mag size to 20.
/:cl:

### CODE REVIEW

File: code/modules/cargo/packs/ammo.dm [hunk 1]
```diff
@@ -75,17 +75,15 @@
 
 /datum/supply_pack/ammo/buckshot
 	name = "Buckshot Crate"
-	desc = "Contains two boxes of buckshot for use in lethal persuasion."
+	desc = "Contains a box of buckshot for use in lethal persuasion,contains 25 shells."
```
Comment (Skrem7) [inline code/modules/cargo/packs/ammo.dm]:
```suggestion
	desc = "Contains a box of twenty-five buckshot shells for use in lethal persuasion."
```

File: code/modules/cargo/packs/ammo.dm [hunk 2]
```diff
@@ -75,17 +75,15 @@
...
 	cost = 2000
-	contains = list(/obj/item/storage/box/lethalshot,
-					/obj/item/storage/box/lethalshot)
+	contains = list(/obj/item/ammo_box/a12g)
 
 /datum/supply_pack/ammo/slugs
 	name = "Shotgun Slug Crate"
-	desc = "Contains two boxes of slug shells for use in lethal persuasion."
+	desc = "Contains a box of slug shells for use in lethal persuasion,contains 25 shells."
```
Comment (Skrem7) [inline code/modules/cargo/packs/ammo.dm]:
```suggestion
	desc = "Contains a box of twenty-five slug shells for use in lethal persuasion."
```

File: code/modules/cargo/packs/ammo.dm [hunk 3]
```diff
@@ -188,3 +186,37 @@
 					/obj/item/ammo_box/a762,
 					/obj/item/ammo_box/a762)
 	cost = 1000
+
+/datum/supply_pack/ammo/a762_ammo_box
+	name = "7.62x39mm Ammo Box Crate"
+	desc = "Contains a 7.62x39mm box for rifles like the SVG-67,contains 60 rounds."
```
Comment (Skrem7) [inline code/modules/cargo/packs/ammo.dm]:
```suggestion
	desc = "Contains a sixty-round 7.62x39mm box for rifles like the SVG-67."
```

File: code/modules/cargo/packs/ammo.dm [hunk 4]
```diff
@@ -188,3 +186,37 @@
...
+	contains = list(/obj/item/ammo_box/a762_39)
+	cost = 1750
+
+/datum/supply_pack/ammo/c556mmHITP_ammo_box
+	name = "5.56 Caseless Ammo Box Crate"
+	desc = "Contains two 5.56mm caseless boxes for SolGov sidearms like the Pistole C,contains 100 rounds total."
```
Comment (Skrem7) [inline code/modules/cargo/packs/ammo.dm]:
```suggestion
	desc = "Contains two fifty-round 5.56mm caseless boxes for SolGov sidearms like the Pistole C, for a total of 100 rounds."
```

File: code/modules/cargo/packs/ammo.dm [hunk 5]
```diff
@@ -188,3 +186,37 @@
...
+	contains = list(/obj/item/ammo_box/c556mmHITP,
+					/obj/item/ammo_box/c556mmHITP)
+	cost = 1500
+
+/datum/supply_pack/ammo/c45_ammo_box
+	name = ".45 Ammo Box Crate"
+	desc = "Contains two .45 boxes for pistols and SMGs like the M1911,contains 100 rounds total."
```
Comment (Skrem7) [inline code/modules/cargo/packs/ammo.dm]:
```suggestion
	desc = "Contains two fifty-round .45 boxes for pistols and SMGs like the M1911 or the C-20r, for a total of 100 rounds."
```

File: code/modules/cargo/packs/ammo.dm [hunk 6]
```diff
@@ -188,3 +186,37 @@
...
+	contains = list(/obj/item/ammo_box/c45,
+					/obj/item/ammo_box/c45)
+	cost = 1500
+
+/datum/supply_pack/ammo/c10mm_ammo_box
+	name = "10mm Ammo Box Crate"
+	desc = "Contains two 10mm boxes for pistols like the Stechkin,contains 100 rounds total."
```
Comment (Skrem7) [inline code/modules/cargo/packs/ammo.dm]:
```suggestion
	desc = "Contains two fifty-round 10mm boxes for pistols and SMGs like the Stechkin or the SkM-44(k), for a total of 100 rounds."
```

File: code/modules/cargo/packs/ammo.dm [hunk 7]
```diff
@@ -188,3 +186,37 @@
...
+	contains = list(/obj/item/ammo_box/c10mm,
+					/obj/item/ammo_box/c10mm)
+	cost = 1500
+
+/datum/supply_pack/ammo/c9mm_ammo_box
+	name = "9mm Ammo Box Crate"
+	desc = "Contains two 9mm boxes for pistols like the Commander or Commisioner,contains 100 rounds total."
```
Comment (Skrem7) [inline code/modules/cargo/packs/ammo.dm]:
```suggestion
	desc = "Contains two fifty-round 9mm boxes for pistols and SMGs such as the Commander or Saber, for a total of 100 rounds."
```

File: code/modules/projectiles/boxes_magazines/external/rifle.dm [hunk 8]
```diff
@@ -60,7 +60,7 @@
 
 /obj/item/ammo_box/magazine/ak47
 	name = "assault rifle magazine (7.62x39mm)"
-	desc = "A sharply-curved, 20-round magazine for 7.62x39mm assault rifles. These rounds do good damage with good armor penetration."
```
Comment (Skrem7) [inline code/modules/projectiles/boxes_magazines/external/rifle.dm]: I believe twenty rounds was the intended number, it's just that max_ammo wasn't updated. This makes it not a strict upgrade over the P-16. This, however, would technically be a balance change.

File: code/modules/cargo/packs/ammo.dm [hunk 9]
```diff
@@ -167,8 +165,8 @@
 	cost = 2000
 
 /datum/supply_pack/ammo/ak47_ammo
-	name = "AKM 7.62x39mm FMJ Magazine Crate"
-	desc = "Contains two 7.62x39mm FMJ magazines for the AKM rifle, each containing twenty rounds."
+	name = "SVG 7.62x39mm FMJ Magazine Crate"
+	desc = "Contains two 7.62x39mm FMJ magazines for the SVG rifle, each containing thirty rounds."
```
Comment (SomeguyManperson) [inline code/modules/cargo/packs/ammo.dm]:
```suggestion
	desc = "Contains two 7.62x39mm FMJ magazines for the SVG rifle, each containing twenty rounds."
```

**Comment by Martinpachu** [comment]
this is probably ready.

**Comment by thgvr** [comment]
These prices seem excessive, what about 500 for each of them?

**Comment by Martinpachu** [comment]
@thgvr nerfed em,how are they looking now?

**Pull request merged**